Data/Ball.cs

```diff
@@ -0,0 +1,18 @@
+namespace Data
+{
+    public class Ball
+    {
+        public int X { get; set; }
```

Comment:
I think XPos and YPos gives more information and thats how its called in frontEnd